lola.html

```diff
@@ -0,0 +1,875 @@
+          All other access control or privilege metadata
+        </li>
+        <li>Breadcrumbs from prior moves
+        </li>
+      </ul>
+
+
+      <p>
+        If the source server has has ‘likes’ information on objects, it should include that collection in each object as per <a href="https://www.w3.org/TR/activitypub/#likes">https://www.w3.org/TR/activitypub/#likes</a>. If the source server has ‘shares’ information, that is included as per <a href="https://www.w3.org/TR/activitypub/#shares">https://www.w3.org/TR/activitypub/#shares</a>.  The source server MAY limit the size of particularly large likes and shares collections.
+      </p>
+
+      <p>
+        Open Issues:  Is there a way to negotiate what extended features are recognized and whether the source should downgrade structured info?  Is there already a way to request posts in one format or another? 
+      </p>
+
+      <h3>Fetching Actor information</h3>
+
+      <p>
+        The Following collection as per <a href="https://www.w3.org/TR/activitypub/#following">https://www.w3.org/TR/activitypub/#following</a>  SHOULD be provided on the Actor object when accessed with the account migration authorization token.  
+      </p>
+
+      <p>
+        If the server does blocking, the personal block list SHOULD be fetchable at the URL advertised on the Actor object, as per <a href="https://codeberg.org/fediverse/fep/src/branch/main/fep/c648/fep-c648.md">https://codeberg.org/fediverse/fep/src/branch/main/fep/c648/fep-c648.md</a>
+      </p>
+
+      <p>
+        Destinations are not required to port and reconstruct Following and Blocked data, as it may not be relevant depending on the use case. If the destination does port and reconstruct this data, it’s probably good to offer it separately from copying content. 
+      </p>
+
+      <p>
+        The destination SHOULD copy the “Liked” collection on the Actor, which the source SHOULD provide if there is an account migration authorization token.  No changes should need to be made and dates and references SHOULD not be made.
+      </p>
+
+      <p>
+        A destination server MAY copy additional Actor information if it’s not overwriting choices already made by the user.  For example, if a new account on a destination does not have an avatar, and the user then chooses to migrate content or follows from another location that does have an avatar, it would be great to copy the avatar too.
+      </p>
+
+      <h3>Group Membership</h3>
+
+      <p>
+        Open Issue: should we add a group membership collection as a separate spec? It has applicability outside of account migration/export/import use cases.
+      </p>
+
+      <h3>Saving “new” copied object</h3>
+
+      <h4>Types</h4>
+
+      <p>
+        Note objects should be presented directly in the destination outbox after migration.  TBD - we need to confirm if this avoids triggering UX notifications on 3rd party servers.
+      </p>
+
+      <h4>Changing Object IDs</h4>
```

Comment:
I think because of the way ActivityPub uses IDs, this is a MUST. The copied item has to have a different ID from its original value.